src/Magnum/Math/Geometry/Test/CMakeLists.txt

```diff
@@ -25,3 +25,7 @@
 
 corrade_add_test(MathGeometryDistanceTest DistanceTest.cpp LIBRARIES MagnumMathTestLib)
 corrade_add_test(MathGeometryIntersectionTest IntersectionTest.cpp LIBRARIES MagnumMathTestLib)
+
+set_property(TARGET
+    MathGeometryDistanceTest
+    APPEND PROPERTY COMPILE_DEFINITIONS "CORRADE_GRACEFUL_ASSERT")
```

Comment:
Ah, there it was because I was setting one property for multiple targets -- easier than writing `target_compile_definitions()` ten times.